resources/views/layouts/menu.blade.php

```diff
@@ -25,8 +25,26 @@
                                                  aria-hidden="true"></i> <span
                             class="hidden-sm">@lang('Overview')</span></a>
                     <ul class="dropdown-menu multi-level" role="menu">
-                        <li><a href="{{ url('overview') }}"><i class="fa fa-tv fa-fw fa-lg"
-                                                               aria-hidden="true"></i> @lang('Dashboard')</a></li>
+                        <li class="dropdown-submenu">
+                            <a><i class="fa fa-tv fa-fw fa-lg" aria-hidden="true"></i> @lang('Dashboard')</a>
+                            <ul class="dropdown-menu">
+                                @foreach($dashboards as $dashboard)
+                                <li><a href="overview?dashboard={{ $dashboard->dashboard_id }}"><i class="fa fa-tv fa-fw fa-lg" aria-hidden="true"></i> {{ $dashboard->dashboard_name }}</a></li>
+                                @endforeach
+                                <li role="presentation" class="divider"></li>
+                                <li>
+                                    <a href="{{ url('toggle_dashboard_editor') }}">
+                                    <i class="fa fa-bar-chart fa-fw fa-lg" aria-hidden="true"></i>
+                                    @if ($hide_dashboard_editor) @lang('Show Dashboard Editor')
+                                    @else @lang('Hide Dashboard Editor')
+                                    @endif</a>
+                                    <form id="toggle_dashboard_editor-form" action="{{ url('toggle_dashboard_editor') }}" method="POST" style="display: none;">
```

Comment:
What does this empty form do?

Reply:
it calls a controller method, similar to:
```
<form id="logout-form" action="{{ route('logout') }}" method="POST" style="display: none;">
{{ csrf_field() }}
</form>
```

Reply:
ok, a little bit confused. Before i didn't get it to work without it, now i tried it without it and worked. ... removed now :-)